app/templates/partials/summary/collapsible-summary.html

```diff
@@ -1,19 +1,19 @@
-<div class="accordion js-accordion">
-    <div class="accordion__controls">
-        <button class="btn btn--secondary btn--small accordion__control js-accordion-toggle-all u-wa--@xs" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Show all" data-close-all-label="{{ _("Hide all") }}" data-open-all-label="{{ _("Show all") }}" data-open="false" aria-hidden="true">{{ _("Show all") }}</button>
+<div class="collapsible js-collapsible">
+    <div class="collapsible__controls">
+        <button class="btn btn--secondary btn--small collapsible__control js-collapsible-toggle-all u-wa--@xs" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Show all" data-ga-label="Show all" data-close-all-label="{{ _('Hide all') }}" data-open-all-label="{{ _('Show all') }}" aria-hidden="true">{{ _("Show all") }}</button>
     </div>
 
-    <div class="js-accordion-content">
+    <div class="js-collapsible-content">
         {%- for group in content.summary.groups -%}
             {%- if group.title -%}
             {%- set outer_loop = loop -%}
-                <h3 class="accordion__title js-accordion-title u-cf icon--accordion" data-js-accordion-event-label="{{group.title}}">
-                    <span class="accordion__title-text">{{group.title}}</span>
-                    <button class="btn btn--secondary btn--small accordion__title-right accordion-unhide@m js-accordion-toggle u-wa--@xs" id="{{ group.id }}-button" data-close-label="{{ _("Hide") }}" data-open-label="{{ _("Show") }}" aria-hidden="true">{{ _("Show") }}</button>
+                <h3 class="collapsible__title js-collapsible-title u-cf icon--collapsible" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Open panel" data-ga-label="{{group.title}}">
+                    <span class="collapsible__title-text">{{group.title}}</span>
+                    <button class="btn btn--secondary btn--small collapsible__title-right js-collapsible-toggle u-wa--@xs" id="{{ group.id }}-button" data-close-label="{{ _('Hide') }}" data-open-label="{{ _('Show') }}" aria-hidden="true" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Open panel" data-ga-label="{{group.title}}">{{ _("Show") }}</button>
```

Comment:
Should be consistent. ie some are `{{ 'Show' }}` some are `{{ "Show" }}`. 

Also some files don't have new lines at the end now, nice to keep it all same.